Extension/package.nls.json

```diff
@@ -928,6 +928,7 @@
     "c_cpp.taskDefinitions.name.description": "The name of the task.",
     "c_cpp.taskDefinitions.command.description": "The path to either a compiler or script that performs compilation.",
     "c_cpp.taskDefinitions.args.description": "Additional arguments to pass to the compiler or compilation script.",
+    "c_cpp.taskDefinitions.fragments.description": "Additional arguments to pass to the compiler that are expected to receive shell processing. For example, '*.c' should expand to all .c files in the working directory.",
```

Comment:
The Windows command prompt doesn't do wildcard expansion. I'd omit the example. 

How about? :  `Command line fragments to pass to the compiler or compilation script. These arguments will receive limited shell command line processing. Multiple arguments may be specified per value.`

And maybe update `args` to, `Arguments to pass to the compiler or compilation script. These are not expected to contain shell escaping and will not receive shell processing. A single argument should be specified per value`.

(The use of `Additional` in the descriptions seems unnecessary to me.)